front-end/packages/robustar/src/views/ImageAnnotation.vue

```diff
@@ -8,15 +8,26 @@
         <div class="d-flex justify-center mb-8">
           <v-btn
             depressed
-            color="warning"
             class="mr-4"
+            @click="
+              $router.push({
+                name: 'ImageList',
+                params: { split: $route.params.split },
+              })
+            "
+          >
+            <v-icon left>mdi-arrow-left</v-icon>Back
```

Comment:
The back button doesn't seem to work. It is not modifying the URL correctly: it goes back to the base url instead of `image-list/split`.